agrirouter-sdk-dotnet-standard-api/Dto/Onboard/OnboardRequest.cs

```diff
@@ -11,7 +11,7 @@ public class OnboardRequest
         ///     External ID.
         /// </summary>
         [JsonProperty(PropertyName = "id")]
-        public string ExternalId { get; set; }
+        public string Uuid { get; set; }
```

Comment:
Why rename this from (the semantically better) `ExternalId` to `Uuid`, when this isn't even the 'real' upstream name?